packages/core/ios/Sources/DdSdk.swift

```diff
@@ -43,7 +43,7 @@ class RNDdSdk: NSObject {
                 // Initializing the SDK twice results in Global.rum and
                 // Global.sharedTracer to be set to no-op instances
                 consolePrint("Datadog SDK is already initialized, skipping initialization.")
-                return
+                resolve(nil)
```

Comment:
shouldn't we still have a `return` in addition to `resolve(nil)`? Otherwise the execution of this method will continue and we will call `Datadog.initialize` and one more time `resolve(nil)`.